### PR TITLE
updated documentation with intermediary information

### PR DESCRIPTION
**Pull request by Phemus**
Updated schemas to reflect latest system changes. This includes:

- Removing gender from identity checks
- Adding identity_checked to get deed
- removing endpoints from organisation schema that is not exposed to customers

Added section explaining both flows for confirming borrower identity (wording has been run past John and agreed)

added section explaining how to assign a proxy and the impact this has on the journey

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,6 +77,86 @@ <h4>
...
           </h4>
         </div>
+        <div class="schemas">
+          <a name="identityChecks"><h2>Identity Checks</h2></a>
+          <p>
+            Under section 111 of the Network Access Agreement HM Land Registry requests conveyancers to confirm
+            that the borrowers identity has been established.
+          </p>
+          <p>
+            Depending on the buisiness model of your organisation there are two methods you can use to confirm that
```
Comment (stuartmccoll) [inline index.html]: Typo in 'business'.

File: index.html [hunk 2]
```diff
@@ -75,6 +77,86 @@ <h4>
...
+            due dilligence has been performed and the borrowers identity has been established.
```
Comment (stuartmccoll) [inline index.html]: Typo in 'diligence'.

File: index.html [hunk 3]
```diff
@@ -75,6 +77,86 @@ <h4>
...
+          The identity-checked endpoint is documented in the
+          <a href="https://raw.githubusercontent.com/LandRegistry/sign-your-mortgage-documentation/gh-pages/schema.yaml">
+            schema
+          </a>.
+        </p>
+        <p>
+          <b>*NOTE*</b> The borrowers will not be able to sign the deed until the identity checks have been successfully
+          carried out either on creation or by using the identity-checked endpoint.
+        </p>
+        <p><b>BY DOING EITHER OF THE STEPS ABOVE YOU ARE CONFIRMING THAT DUE DILLIGENCE HAS BEEN CARRIED OUT AND
```
Comment (stuartmccoll) [inline index.html]: Typo in 'diligence'.

File: index.html [hunk 4]
```diff
@@ -75,6 +77,86 @@ <h4>
...
+            in the format it is provided by the organisations endpoint.
+          </p>
+          <p>
+            For more information on consuming the organisations functionality see the schema file for this endpoint
+            <a href="https://raw.githubusercontent.com/LandRegistry/sign-your-mortgage-documentation/gh-pages/organisation-api-schema.yaml">
+              here
+            </a>.
+          </p>
+            <p>
+              <b>*NOTE*</b> IF you set a value in the "proxy_organisation_name" field then only this organisation will
```
Comment (stuartmccoll) [inline index.html]: 'If' is all uppercased here, not sure if you intended that?

File: index.html [hunk 5]
```diff
@@ -266,6 +349,40 @@ <h5>Responses</h5>
...
+          <h5>Payload</h5>
+          <pre>
+            <code>
+{
+  "borrowers": [
+    {
+      "id": 1108,
+      "forename": "Ann",
+      "surname": "Smith",
+      "gender": "Female",
```
Comment (stuartmccoll) [inline index.html]: We can drop the `gender` key here for both borrowers.

File: index.html [hunk 6]
```diff
@@ -266,6 +349,40 @@ <h5>Responses</h5>
...
+
+    }
+  ]
+}
+            </code>
+          </pre>
+          <h5>Responses</h5>
+          200 OK
+          <pre>
+            <code>
```
Comment (stuartmccoll) [inline index.html:381]: Did you mean to put something in the `<code>` block?

File: index.html [hunk 7]
```diff
@@ -75,6 +77,86 @@ <h4>
...
+          </a>.
+        </p>
+        <h3>If the checks will be performed by a different organisation than the one that created the deed</h3>
+        <p>
+          If the checks will be carried out by a different organisation than the one creating the deed then on deed
+          creation the identity_checked value should be set to "N".
+        </p>
+        <p>
+          The organisation carrying out the checks will be able to perform the identity checks by calling the
+          /identity-checked endpoint on the system and providing the name address and date of birth for each borrower.
```
Comment (stuartmccoll) [inline index.html]: And the ID?

File: organisation-api-schema.yaml [hunk 8]
```diff
@@ -33,57 +33,8 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Organisation_List'
-  '/organisations/{organisation_id}':
```
Comment (stuartmccoll) [inline organisation-api-schema.yaml:36]: Just double checking, I'm assuming you've taken it out as the conveyancer won't be able to hit this through gatekeeper?

**Review by stuartmccoll — CHANGES_REQUESTED** [review]
A few typos, and `gender` key left in the example payload for the `/identity-checks` endpoint.

File: organisation-api-schema.yaml (hunk 8 above)
Comment (Phemus) [reply organisation-api-schema.yaml:36]: yes I removed all endpoints not exposed to conveyancers

**Review by stuartmccoll — APPROVED** [review]
👍

**Pull request merged**